doc/examples.md

```diff
@@ -11,3 +11,7 @@
 
 	Repository for templates for haskell projects, including some templates using
 	**servant**. These templates can be used with `stack new`.
+
+- ** [custom-monad](https://github.com/themoritz/diener)**:
+
+        A custom monad that can replace `IO` in servant applications. It adds among other things logging functionality and a reader (for database conncetions). A full usage example of servant/diener is also provided.
```

Comment:
`conncetions` -> `connections`.
`a reader` -> `a reader monad`?


Could you also line-break at around 80 chars?